pkg/build/builder/daemonless.go

```diff
@@ -57,8 +126,25 @@ func pullDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 		return fmt.Errorf("error parsing image name to pull %s: %v", "docker://"+imageName, err)
 	}
 
+	dstFile, err := ioutil.TempFile("", "config")
+	if err != nil {
+		return fmt.Errorf("error create tmp credentials file: %v", err)
```

Comment:
s/create/creating/

also, i would pick either "temp" or "tmp" and stick with it throughout your pull request.

Reply:
Thanks. Fixed.